### PR TITLE
[mod_voicemail] add option to skip goodbye

### PR DESCRIPTION
**Pull request by wmasilva**
add channel variable to skip goodbye message when leaving a message:

```
<action application="set" data="voicemail_skip_goodbye=true"/>
...

```

### CODE REVIEW

**Review by dragos-oancea — DISMISSED** [review]

**Review by Fernandojdk — DISMISSED** [review]

File: src/mod/applications/mod_voicemail/mod_voicemail.c [hunk 1]
```diff
@@ -3402,11 +3402,13 @@ static switch_status_t voicemail_leave_main(switch_core_session_t *session, vm_p
 	switch_bool_t skip_greeting = switch_true(switch_channel_get_variable(channel, "skip_greeting"));
 	switch_bool_t skip_instructions = switch_true(switch_channel_get_variable(channel, "skip_instructions"));
 	switch_bool_t skip_record_urgent_check = switch_true(switch_channel_get_variable(channel, "skip_record_urgent_check"));
+	switch_bool_t voicemail_skip_goodbye = switch_true(switch_channel_get_variable(channel, "voicemail_skip_goodbye"));
```
Comment (andywolk) [inline src/mod/applications/mod_voicemail/mod_voicemail.c:3405]: Please use `switch_channel_var_true()` instead.

**Review by andywolk — CHANGES_REQUESTED** [review]

File: src/mod/applications/mod_voicemail/mod_voicemail.c (hunk 1 above)
Comment (wmasilva) [reply src/mod/applications/mod_voicemail/mod_voicemail.c:3405]: sure, also the other variables?

Comment (andywolk) [reply src/mod/applications/mod_voicemail/mod_voicemail.c:3405]: makes sense

Comment (wmasilva) [reply src/mod/applications/mod_voicemail/mod_voicemail.c]: done for all.

**Review by andywolk — APPROVED** [review]

**Comment by andywolk** [comment]
First comment updated to reflect actual change `voicemail_skip_goodbye`

**Pull request merged**